crates/sb_ai/lib.rs

```diff
@@ -218,3 +217,45 @@ pub async fn op_sb_ai_run_model(
         bail!("model not supported")
     }
 }
+
+#[op2]
+#[serde]
+pub fn op_sb_ai_init_pipeline(
+    state: &mut OpState,
+    #[string] task: String,
+    #[string] name: String,
+) -> Result<(), AnyError> {
+    let name = Option::from(name).filter(|name| !name.is_empty());
+
+    match task.trim() {
+        "feature-extraction" => init_feature_extraction(state, task, name),
+        _ => return Err(anyhow!("Not supported pipeline task: {task}")),
```

Comment:
This might be an unnecessary `return` I think.

Reply:
resolved at https://github.com/supabase/edge-runtime/pull/368/commits/9d0d0db14de52ef5ed88d05cddb8590af11347fa